webapp/src/js/services/resource-icons.js

```diff
@@ -57,19 +57,23 @@ angular.module('inboxServices').factory('ResourceIcons',
 
     const getHtml = (name, docId) => {
       const image = getHtmlContent(name, docId);
-      return `<span class="${CSS_CLASS[DOC_IDS.indexOf(docId)]}" title="${name}">${image}</span>`;
+      // Handle title attribute for branding doc specially
+      // https://github.com/medic/medic/issues/5531
+      if (docId === DOC_IDS[1]) {
+        return `<span class="${CSS_CLASS[1]}" data-title="${name}">${image}</span>`;
+      } else {
+        return `<span class="${CSS_CLASS[DOC_IDS.indexOf(docId)]}" title="${name}">${image}</span>`;
+      }
```

Comment:
There's quite a lot going on here and there's a bit of duplication. I think it would be clearer if it were written like...

```js
const titleAttribute = docId === DOC_IDS[1] ? 'data-title' : 'title';
const className = CSS_CLASS[DOC_IDS.indexOf(docId)];
return `<span class="${className}" ${titleAttribute}="${name}">${image}</span>`;
```

What do you think?

Reply:
Couldn't agree more